src/rebar_coveralls.erl

```diff
@@ -72,18 +73,32 @@ do_coveralls(ConvertAndSend, Get, GetLocal) ->
              "need to specify coveralls_* and cover_export_enabled "
              "in rebar.config"});
     false ->
-      io:format("rebar_coveralls:"
-                "Exporting cover data from ~s using service ~s and jobid ~s~n",
-                [File, ServiceName, ServiceJobId]),
-      ok = ConvertAndSend(File, ServiceJobId, ServiceName)
+      DoCoveralls = (GetLocal(do_coveralls_after_ct, true) andalso Task == ct) orelse 
+                    (GetLocal(do_coveralls_after_eunit, true) andalso Task == eunit) orelse
+                    Task == 'send-coveralls',
+      case DoCoveralls of
+        true ->
+          io:format("rebar_coveralls:"
+                    "Exporting cover data from ~s using service ~s and jobid ~s~n",
+                    [File, ServiceName, ServiceJobId]),
+          ok = ConvertAndSend(File, ServiceJobId, ServiceName);
+        _ -> maybe_skip()
+      end
   end.
 
+
+-ifdef(TEST).
```

Comment:
make the maybe function a parameter to do_coveralls instead to follow the coding style used in the rest of the project